### DISCUSSION TITLE
if material supports geojson

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#5167] [Q&A] [answered]
It looks like [github](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams) support geojson, I wonder if material is also supporting this
```geojson
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "id": 1,
      "properties": {
        "ID": 0
      },
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
              [-90,35],
              [-90,30],
              [-85,30],
              [-85,35],
              [-90,35]
          ]
        ]
      }
    }
  ]
}
```

**Comment by squidfunk** [comment, marked as answer]
There's currently no support for GeoJSON and it's also not planned. However, it should be possible to add this yourself via [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript) and a [custom fence](https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#custom-fences), similar to how we implemented support for Mermaid.js.

**Reply by xihajun** [reply]
thanks!
